src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -233,48 +245,60 @@ public partial struct JsonWriterOptions
+        public readonly System.Buffers.ReadOnlySequence<byte> ValueSequence { get { throw null; } }
+        public readonly System.ReadOnlySpan<byte> ValueSpan { get { throw null; } }
         public bool GetBoolean() { throw null; }
+        public byte GetByte() { throw null; }
         public byte[] GetBytesFromBase64() { throw null; }
         public string GetComment() { throw null; }
         public System.DateTime GetDateTime() { throw null; }
         public System.DateTimeOffset GetDateTimeOffset() { throw null; }
         public decimal GetDecimal() { throw null; }
         public double GetDouble() { throw null; }
         public System.Guid GetGuid() { throw null; }
+        public short GetInt16() { throw null; }
         public int GetInt32() { throw null; }
         public long GetInt64() { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public sbyte GetSByte() { throw null; }
         public float GetSingle() { throw null; }
         public string GetString() { throw null; }
         [System.CLSCompliantAttribute(false)]
+        public ushort GetUInt16() { throw null; }
+        [System.CLSCompliantAttribute(false)]
         public uint GetUInt32() { throw null; }
         [System.CLSCompliantAttribute(false)]
         public ulong GetUInt64() { throw null; }
         public bool Read() { throw null; }
         public void Skip() { }
+        public bool TryGetByte(out byte value) { throw null; }
         public bool TryGetBytesFromBase64(out byte[] value) { throw null; }
         public bool TryGetDateTime(out System.DateTime value) { throw null; }
         public bool TryGetDateTimeOffset(out System.DateTimeOffset value) { throw null; }
         public bool TryGetDecimal(out decimal value) { throw null; }
         public bool TryGetDouble(out double value) { throw null; }
         public bool TryGetGuid(out System.Guid value) { throw null; }
+        public bool TryGetInt16(out short value) { throw null; }
         public bool TryGetInt32(out int value) { throw null; }
         public bool TryGetInt64(out long value) { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public bool TryGetSByte(out sbyte value) { throw null; }
         public bool TryGetSingle(out float value) { throw null; }
         [System.CLSCompliantAttribute(false)]
+        public bool TryGetUInt16(out ushort value) { throw null; }
+        [System.CLSCompliantAttribute(false)]
         public bool TryGetUInt32(out uint value) { throw null; }
         [System.CLSCompliantAttribute(false)]
         public bool TryGetUInt64(out ulong value) { throw null; }
         public bool TrySkip() { throw null; }
         public bool ValueTextEquals(System.ReadOnlySpan<byte> utf8Text) { throw null; }
         public bool ValueTextEquals(System.ReadOnlySpan<char> text) { throw null; }
-        public bool ValueTextEquals(string text) { throw null; }
+        public bool ValueTextEquals(string utf8Text) { throw null; }
```

Comment:
This name change seems bad.  Presumably the ref got manually corrected at some point and the src is out of date.  (Or the src got changed after the last ref generation)

The correct name here is `text`, as System.String isn't UTF-8.

Reply:
The discrepancy of parameter name between src and ref has been since introduction of the new API. Reference: https://github.com/dotnet/corefx/commit/f09135a373caa94eca1566f7ad497aaf19587140

I will revert here and also change the parameter name in the API so that ref code auto-generated in future won't have this issue.

Reply:
The same issue came up here: https://github.com/dotnet/corefx/pull/38409#discussion_r294029941

> I will revert here and also change the parameter name in the API so that ref code auto-generated in future won't have this issue.

That would be awesome! I was just about to submit a PR for this change, but I will wait for you to update it. Thanks.